.vuepress/config.js

```diff
@@ -124,7 +124,7 @@ module.exports = {
         },
         {
           title: '📕 Cookbooks',
-          path: '/learn/cookbooks/',
+          path: '/learn/cookbooks/running_production',
```

Comment:
I think cookbook can keep its landing page with a bullet point of every cookbook. 
Since when clicking on `cookbooks` it opens the sub-navbar of `Run in production` it become harder to see the other ones.